src/common/components/FieldGroup/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from "react";
+
+import classNames from "classnames/bind";
+import styles from "./Dropdown.module.scss";
+
+import { Input, SuffixElement } from "../parts";
+import { ClassNameCSSProperties, DropdownProps } from "../type";
+
+const cn = classNames.bind(styles);
+
+// interface DropdownProps {
+//   options?: string[];
+//   onClick?: (option: string) => void;
+//   name?: string;
+//   placeholder?: string;
+//   size?: "sm" | "md";
+//   color?: "white" | "gray";
+//   className?: string;
+// }
+
+// interface ClassNameCSSProperties extends CSSProperties {
+//   "--width"?: string;
+// }
+
+export default function Dropdown({
+  options,
+  onClick,
+  name,
+  placeholder,
+  size = "md",
+  color = "white",
+  className,
+}: DropdownProps) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedOption, setSelectedOption] = useState("");
+
+  const handleDropdownClick = () => {
+    setIsOpen((prevIsOpen) => !prevIsOpen);
+  };
+
+  const handleOptionSelect = (option: string) => {
+    setSelectedOption(option);
+    setIsOpen((prevIsOpen) => !prevIsOpen);
```

Comment:
드랍다운이 열려있는 상태라서 닫히겠지만, 명시적으로 표현해줘도 좋을 것 같습니다
```suggestion
    setIsOpen(false);
```